internal/storage/v1/elasticsearch/options.go

```diff
@@ -424,6 +430,14 @@ func DefaultConfig() config.Configuration {
 				Username: "",
```

Comment:
these are all zero values, no need to assign them.

If you switch to use the Optional type when you will need to use WithDefault() and provide the default for the reload interval, which is the only non-zero value.